### ISSUE TITLE
Add Cookie consent modal

### ISSUE THREAD

**Issue by peterjurco** [api3dao/api3-dao-dashboard#467]
As preparation for adding Google Analytics and Clarity to the DAO dashboard we want to add user consent for analytical cookies (along the error reporting ones) to the DAO Dashboard.  Clicking on "Manage Settings" user should see the Privacy settings modal similar to the one on the Market. This should apply only to the new rebranded version. 

Design:  https://www.figma.com/design/NG6QClnVrvPRZ5gJ3s0zen/%E2%9F%81--Api3-DAO---Branding-Update?node-id=9704-161128&node-type=frame&m=dev

![Image](https://github.com/user-attachments/assets/b5644dca-f493-432e-86c7-7ca0df1463ce)

![Image](https://github.com/user-attachments/assets/469eec8b-fb65-49b1-a4a8-50a4afc2d008)

**Comment by PrenaamD** [comment]
@Anboias 
According to the [design](https://www.figma.com/design/NG6QClnVrvPRZ5gJ3s0zen/%E2%9F%81--Api3-DAO---Branding-Update?node-id=8676-107914&t=yptzSdV568YoukP8-1) the cookie banner should read
`In order to provide the best services for you, we collect anonymized error data through [Sentry](https://sentry.io/) and use analytics cookies to improve our products.` 
The implementation's reads `In order to provide the best services for you, we collect anonymized error data through [Sentry](https://sentry.io/). We do not gather IP address or user agent information.`

**Comment by Anboias** [comment]
@PrenaamD, yep. Copied it 1on1 from the Market, felt like that's the best way to go. I'll ask the team about it.

**Comment by Anboias** [comment]
@PrenaamD 
I was actually looking in the wrong place, my bad. You were right. Corrected.
